src/mo_pack/_packing.pyx

```diff
@@ -51,13 +49,14 @@ cdef extern from "wgdosstuff.h":
                     function* parent) nogil
 
 
-cdef void MO_syslog(int value, char* message, const function* const caller):
+
+cdef void MO_syslog(int value, char* message, const function* const caller) noexcept:
```

Comment:
See https://cython.readthedocs.io/en/latest/src/userguide/migrating_to_cy30.html#exception-values-and-noexcept